extra/patterns/equilateral.js

```diff
@@ -0,0 +1,12 @@
+let size;
```

Comment:
No need to declare size variable here, It's unused.

Reply:
ok sir, I'll fix this